testing/test_junitxml.py

```diff
@@ -249,6 +249,19 @@ def test_skip():
         snode = tnode.find_first_by_tag("skipped")
         snode.assert_attr(type="pytest.skip", message="hello25", )
 
+
+    def test_mark_skip_doesnt_capture_output(self, testdir):
+        testdir.makepyfile("""
+            import pytest
+            @pytest.mark.skip(reason="foo")
+            def test_skip():
+                print("bar!")
+        """)
+        result, dom = runandparse(testdir)
+        assert result.ret == 0
+        node = dom.find_first_by_tag("testsuite")
+        assert not node.find_first_by_tag("system-out")
```

Comment:
To be safer, perhaps it would be better to also assert that `"bar!"` doesn't appear on the entire XML text?